src/assets/js/pages/EventPage/index.js

```diff
@@ -265,6 +266,16 @@ class EventPage extends React.Component {
                     </NavLink>
                   </NavItem>
                 ))}
+                {isThirdParty && 
+                  <NavItem className="ml-auto event-tab__alert my-auto">
+                    Third Party Event
+                    <span className="m-2">
+                      <FA name="question-circle" id={'ThirdPartyTooltip'}></FA>
+                      <UncontrolledTooltip placement="right" target={'ThirdPartyTooltip'}>
+                        This Event Is Not Run By TESC. 
```

Comment:
Capitalisation is hard to read.
`This event is not run by TESC` is easier to read